elastic/indexer/insta_face/main/main.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/elastic/go-elasticsearch/v7"
+	"github.com/elastic/go-elasticsearch/v7/esutil"
+
+	elasticsearch_inserter "github.com/codeuniversity/smag-mvp/elastic/indexer"
```

Comment:
let's also call this indexer:
```suggestion
	elasticsearch_indexer "github.com/codeuniversity/smag-mvp/elastic/indexer"
```